streampark-flink/streampark-flink-submit/streampark-flink-submit-core/src/main/scala/org/apache/streampark/flink/submit/impl/YarnApplicationSubmit.scala

```diff
@@ -121,16 +121,18 @@ object YarnApplicationSubmit extends YarnSubmitTrait {
 
           val applicationConfiguration = ApplicationConfiguration.fromConfiguration(flinkConfig)
           var applicationId: ApplicationId = null
+          var jobManagerUrl: String = null
           clusterClient = clusterDescriptor.deployApplicationCluster(clusterSpecification, applicationConfiguration).getClusterClient
           applicationId = clusterClient.getClusterId
+          jobManagerUrl = clusterClient.getWebInterfaceURL;
           logInfo(
             s"""
                |-------------------------<<applicationId>>------------------------
                |Flink Job Started: applicationId: $applicationId
                |__________________________________________________________________
                |""".stripMargin)
 
-          SubmitResponse(applicationId.toString, flinkConfig.toMap)
+          SubmitResponse(applicationId.toString, flinkConfig.toMap, "", jobManagerUrl);
```

Comment:
you can improve this code: SubmitResponse(applicationId.toString, flinkConfig.toMap, jobManagerUrl = jobManagerUrl);